python/sx12xx_remove_length.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 jgromes <gromes.jan@gmail.com>
+#
+# This file is part of gr-satellites
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import numpy
+from gnuradio import gr
+import pmt
+
+class sx12xx_remove_length(gr.basic_block):
```

Comment:
Isn't this block exactly the same as [`cc11xx_remove_length`](https://github.com/daniestevez/gr-satellites/blob/master/python/cc11xx_remove_length.py)?

If so, probably the name `cc11xx_remove_length` is a bit unfortunate if it's also going to serve for other chips. In fact, I see that it would be better to have a block that is a bit more general and depends on some parameters.

However, I rather not introduce a new block which is a duplicate of an existing one.

Reply:
As far as I can tell, most sub-GHz FSK chips follow the convention of sync-length-payload-CRC, so perhaps a more generic block to "cut out" a part of the packet based on cut start and cut length would be appropriate. The current X_remove_length implementation assumes we want to remove the first byte of the packet.

I'll replace `sx12xx_remove_length` with `cc11xx_remove_length` in this PR and open a new one to implement the more generic cut block.

Reply:
In fact there is already [`header_remover`](https://github.com/daniestevez/gr-satellites/blob/master/python/header_remover.py), which removes any fixed number `n`of bytes from the header. Thus, `cc11xx_remove_length` is a particular case of `header_remover.py` with `n=1`, so we should also eliminate `cc11xx_remove_length`.

In fact, I think that we need a block (or two) to cut PDUs that has similar "syntax" and functionality of the `head` and `tail` UNIX functions (with the `-c` parameter, I mean). I'll open an issue to reflect this.